From: Selivanov Pavel <selivan.at.github@gmail.com>
Date: Sat, 26 May 2018 01:23:44 +0300
Subject: rsyslog.service.in: move LimitNOFILE to correct section

systemd[1]: [/lib/systemd/system/rsyslog.service:19] Unknown lvalue 'LimitNOFILE' in section 'Install'

(cherry picked from commit e74c7e41addbb36a5900e25a19bd51d91c6a08a8)
---
 rsyslog.service.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/rsyslog.service.in b/rsyslog.service.in
index 03f4eed..8d01249 100644
--- a/rsyslog.service.in
+++ b/rsyslog.service.in
@@ -10,10 +10,10 @@ ExecStart=@sbindir@/rsyslogd -n -iNONE
 StandardOutput=null
 Restart=on-failure
 
-[Install]
-WantedBy=multi-user.target
-Alias=syslog.service
-
 # Increase the default a bit in order to allow many simultaneous
 # files to be monitored, we might need a lot of fds.
 LimitNOFILE=16384
+
+[Install]
+WantedBy=multi-user.target
+Alias=syslog.service
